daemon/cmd/datapath.go

```diff
@@ -334,11 +341,75 @@ func (d *Daemon) initMaps() error {
 		return err
 	}
 
+	pm := probes.NewProbeManager()
+	supportedMapTypes := pm.GetMapTypes()
+	createSockRevNatMaps := option.Config.EnableHostReachableServices &&
+		option.Config.EnableHostServicesUDP && supportedMapTypes.HaveLruHashMapType
 	if err := d.svc.InitMaps(option.Config.EnableIPv6, option.Config.EnableIPv4,
-		option.Config.RestoreState); err != nil {
+		createSockRevNatMaps, option.Config.RestoreState); err != nil {
 		log.WithError(err).Fatal("Unable to initialize service maps")
 	}
 
+	if err := eventsmap.InitMap(); err != nil {
+		return err
+	}
+
+	if err := signalmap.InitMap(); err != nil {
+		return err
+	}
+
+	if err := policymap.InitCallMap(); err != nil {
+		return err
+	}
+
+	if option.Config.EnableIPSec {
+		if err := encrypt.MapCreate(); err != nil {
```

Comment:
This is also created in `pkg/datapath/linux/ipsec.loadIPSecKeys`: https://github.com/cilium/cilium/blob/84dd81c6b235dad0d4e175ae9f032496ef85a814/pkg/datapath/linux/ipsec/ipsec_linux.go#L449-L451

Should probably be removed from there.

Reply:
Thanks for finding this! It looks like the map creation in `ipsec_linux.go` happens before the call to `initMaps()` (via `setupIPSec()` in `NewDaemon()`), so I just need to remove the commit that adds the encrypt map creation to `initMaps()`.